docs/jwst/cube_build/arguments.rst

```diff
@@ -122,3 +129,43 @@ A parameter only used for investigating which detector pixels contributed to a c
 
   The string is the x,y,z value of the cube spaxel that is being investigated. The  numbering starts counting at 0.
   To print information to the screeen about the x = 10, y = 20, z = 35 spaxel the parameter string value is '10 20 35'.
+
+.. _offsets:
+
+Creating an offset file
+-----------------------
+
+The offset file is an ASDF formated file :`<https://asdf-standard.readthedocs.io/>`_ stands for "Advanced Scientific Data. For each
+input file in the spec3 assocation used to build the IFU cubes, the offset files needs to have a corresponding right ascension and declination offset given arc seconds.
+Below is an example of how to make an ASDF offset file. It is assumed the user has determined the
+offsets to apply to the data in each file. The offsets information is stored in three lists:
+`filenames`, `raoffset` and `decoffset`.  The units of the ra and dec offsets 
+are required to be in the offset file and only the unit, `arcsec`, is allowed. The file names should
+not contain the directory path. The offset asdf filename can be any name, but it must have the `asdf` extension.
+Below `num` is the number of files.
+
+
+.. code-block:: python
+		
+   import asdf
+   import astropy.units as u
+   
+   filename = []
+   raoffset = []
+   decoffset = []
+   for i in range(num):
+       filename.append(file[i])
+       raoffset.append(ra_center1[i])
+       decoffset.append(dec_center1[i])
```

Comment:
I would like to the keep the one with the loop because that is how they will be made in general. I think that is straight forward to me. I will also add your example.

Reply:
The reason I asked is that the code in the loop example is not runnable as-is - you'd have to set up the file, ra_center1, dec_center1, and num variables first, and it's not clear how you'd do that.  To keep the example simple, I think it's better to just define the lists.

Reply:
I wrote in the comments that it is up the user to provide those. I can make that clearer.